FLIR/conservator/paginated_query.py

```diff
@@ -61,9 +61,7 @@ def __init__(
         self._total_items = 0
         if reverse:
             if not total_unpack_field:
-                raise KeyError(
-                    f"total_unpack_field must be supplied if reverse is True"
-                )
+                raise KeyError("total_unpack_field must be supplied if reverse is True")
```

Comment:
The `reverse` and `total_unpack_field` parameters are obsolete and could be dropped.

Reply:
Removing those would also involve removing `get_frames_reversed()` from `FLIR/conservator/wrappers/dataset.py` and the test `test_add_get_frames_reversed()` in `test/integration/test_dataset.py`.  Maybe a separate PR?

Reply:
Yeah. I only modified that file because of the f-string. This current PR probably shouldn't be reviewed until the corresponding change to Conservator is merged and deployed to staging - note that Jenkins now uses the Conservator ECR docker image tagged with `staging` instead of `prod` when running integration tests.